src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -14,159 +14,133 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+          <feature name="ALIRO"/>
+        </optionalConform>
+      </feature>
+    </features>
+    <!-- The principle behind  mandatory/optional is the following:
          1. If attribute/command is marked as mandatory it should be mandatory here.
          2. If attribute/command is marked as optional it should be optional here.
          3. Everything that depends on a certain feature is optional because we have no way
             of setting up the dependencies here. Dependencies would be probably resolved in the
             cluster itself. Those attributes/commands are marked with a special comment. -->
-    
     <!-- Attributes -->
-    <attribute side="server" code="0" name="LockState" define="LOCK_STATE" type="DlLockState" min="0" max="3" isNullable="true" reportable="true">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="1" name="LockType" define="LOCK_TYPE" type="DlLockType" min="0" max="11">
-      <mandatoryConform/>
-    </attribute>
-    <attribute side="server" code="2" name="ActuatorEnabled" define="ACTUATOR_ENABLED" type="boolean">
-      <mandatoryConform/>
-    </attribute>
+    <attribute side="server" code="0" name="LockState" define="LOCK_STATE" type="DlLockState" min="0" max="3" isNullable="true"/>
+    <attribute side="server" code="1" name="LockType" define="LOCK_TYPE" type="DlLockType" min="0" max="11"/>
+    <attribute side="server" code="2" name="ActuatorEnabled" define="ACTUATOR_ENABLED" type="boolean"/>
     <!-- Conformance feature DPS - for now optional -->
-    <attribute side="server" code="3" name="DoorState" define="DOOR_STATE" type="DoorStateEnum" min="0" max="5" isNullable="true" reportable="true" optional="true">
+    <attribute side="server" code="3" name="DoorState" define="DOOR_STATE" type="DoorStateEnum" min="0" max="5" isNullable="true" optional="true">
       <mandatoryConform>
         <feature name="DPS"/>
       </mandatoryConform>
     </attribute>
     <!-- Conformance feature [DPS] - for now optional -->
     <attribute side="server" code="4" name="DoorOpenEvents" define="DOOR_OPEN_EVENTS" type="int32u" writable="true" optional="true">
-      <access op="read" role="view" />
       <access op="write" role="manage"/>
       <optionalConform>
         <feature name="DPS"/>
       </optionalConform>
     </attribute>
     <!-- Conformance feature [DPS] - for now optional -->
     <attribute side="server" code="5" name="DoorClosedEvents" define="DOOR_CLOSED_EVENTS" type="int32u" writable="true" optional="true">
-      <access op="read" role="view" />
       <access op="write" role="manage"/>
       <optionalConform>
         <feature name="DPS"/>
       </optionalConform>
     </attribute>
     <!-- Conformance feature [DPS] - for now optional -->
     <attribute side="server" code="6" name="OpenPeriod" define="OPEN_PERIOD" type="int16u" writable="true" optional="true">
-      <access op="read" role="view" />
       <access op="write" role="manage"/>
```

Comment:
The matter xml spec contains the read view level access.
https://github.com/project-chip/connectedhomeip/blob/master/data_model/master/clusters/DoorLock.xml#L739-L760